src/middleware/submissionHandler.js

```diff
@@ -75,8 +75,10 @@ module.exports = (router, resourceName, resourceId) => {
 
       // If this is a get method, then filter the model query.
       if (isGet) {
-        req.countQuery = req.countQuery || req.model || this.model;
-        req.modelQuery = req.modelQuery || req.model || this.model;
+        const submissionModel = hook.alter('getMongoCollection', req);
```

Comment:
Can we call this hook ```submissionCollection```?  Makes more sense since there are a number of different mongo collections.